app.json

```diff
@@ -0,0 +1,59 @@
+{
+  "name": "hostedgpt",
+  "description": "An open version of ChatGPT you can host anywhere or run locally.",
+  "repository": "https://github.com/mgomes/hostedgpt",
```

Comment:
Thanks for putting this PR together! Let’s definitely get it added. In reviewing now. First Q:

How do we handle this line in our forking scenario? Can we point to the main repo or do we need to set this to be a placeholder and then have each person update it with their value.

Or even better, does Heroku support an ENV here?

Reply:
Looks like this was left over from testing. I believe we can remove this completely and also change the deploy button to not specify a `template` parameter at all. I think in that case it uses the referrer. That way it should work with forks.

Just curious, though, why do the docs recommend forking?

Reply:
I’m not certain it’s the right recommendation, but my thinking was a little more control. I configured this repo so that everyone’s fork auto syncs once in the middle of the night. So this way I can merge things into main, test them myself throughout the day, and then at night it pulls into everyone’s fork. I don’t love the idea that every merge I do into main I’m triggering a whole bunch of deploys.

Less importantly it also lets me see who is using the projected. I regularly skim all the forks and sometimes ping people.

The only concern I’ve had is that I’m not 100% about this auto sync and in what situations it might stop working, but so far I think it’s good.